docker-compose-build.yml

```diff
@@ -192,12 +192,14 @@ services:
         environment:
             - DASHDEBUG=0
 
-    prosoul:
-        image: acsdocker/prosoul
+prosoul:
+        image: bitergia/prosoul
```

Comment:
Notice that we testing the fix in #44 , it was with `acsdocker`, not bitergia

Reply:
We moved the image to https://hub.docker.com/r/bitergia/prosoul, since for acsdocker/prosoul we needed Alvaro's credentials